src/repo.rs

```diff
@@ -2100,6 +2101,40 @@ impl Repository {
         }
         Ok(ignored == 1)
     }
+
+    /// Look up the value of one git attribute for path. Equivalent to libgit2::git_attr_get.
+    pub fn get_attr(&self, flags: CheckAttributeFlags, path: &str, name: &str) -> Result<AttributeType, Error> {
+        let mut value_out : *mut c_char = 0 as *mut c_char;
+        let value_out_ptr : *mut *mut c_char = &mut value_out; 
+        
+        let c_flags = flags.bits() as c_uint;
+        let c_path = CString::new(path).expect("The path value contained an invalid 0 byte within the string");
+        let c_name = CString::new(name).expect("The name value contained an invalid 0 byte within the string");
+
+        let attribute_type = unsafe {
+            try_call!(raw::git_attr_get(value_out_ptr, self.raw, c_flags, c_path, c_name));
```

Comment:
Is try_call the correct macro here? I borrowed it from the other usages, but I'm not entirely sure what it provides. It seems to do some auto-conversions?